### PR TITLE
Create initial folder structure

### PR DESCRIPTION
**Pull request by divyanshigkmit**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.env
```
Comment (dhruvildave007) [inline .gitignore:1]: create `.env.sample` file and add only keys which has been used in `.env` file.
for eg.
```
DB_HOST=
DB_USERNAME=
DB_PORT=
DB_DATABASE=
DB_PASSWORD= 
```

File: helper/common-functions.helper.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        console.log(value);
+        return next();
+    }
+
+    const { details } = error;
+    const message = details.map(i => i.message).join(',');
+    return commonErrorHandler(req, res, message, 422);
+};
+
+const getReasonPhrase = function (statusCode) {
```
Comment (dhruvildave007) [inline helper/common-functions.helper.js]: is there any need for this function?

File: config/config.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    host: '127.0.0.1',
+    dialect: 'postgres',
+  },
+  production: {
+    username: "root",
+    password: null,
+    database: "database_production",
+    host: "127.0.0.1",
+    dialect: "mysql",
+  },
```
Comment (dhruvildave007) [inline config/config.js]: use environment variable from .env for production as well.

**Review by dhruvildave007 — CHANGES_REQUESTED** [review]
please do required changes

File: helper/common-functions.helper.js (hunk 2 above)
Comment (divyanshigkmit) [reply helper/common-functions.helper.js]: not now, but it will be used in logging.

Comment (dhruvildave007) [reply helper/common-functions.helper.js]: ok

File: config/config.js (hunk 3 above)
Comment (divyanshigkmit) [reply config/config.js]: done.

File: .gitignore (hunk 1 above)
Comment (divyanshigkmit) [reply .gitignore:1]: done.

**Review by dhruvildave007 — APPROVED** [review]

**Pull request merged**